### ISSUE TITLE
SignalP6 threadlock

### ISSUE THREAD

**Issue by fmobegi** [ccdmb/predector#51]
Hi Darcy,
Getting the following error when running predictor:
```
Error executing process > 'signalp_v6 (2)'

Caused by:
  Process `signalp_v6 (2)` terminated with an error exit status (137)

Command executed:

  mkdir -p tmpdir
  
     run () {
         set -e
         TMPDIR="${PWD}/tmp$$"
         mkdir -p "${TMPDIR}"
  
  export OMP_NUM_THREADS=1
         signalp6           --fastafile "$1"           --output_dir "${TMPDIR}"           --format txt           --organism eukarya           --mode fast         1>&2
  
         cat "${TMPDIR}/prediction_results.txt"
  
         rm -rf -- "${TMPDIR}"
     }
     export -f run
  
     CHUNKSIZE="$(decide_task_chunksize.sh in.fasta "4" "100")"
  
     parallel         --halt now,fail=1         --joblog log.txt         -j "4"         -N "${CHUNKSIZE}"         --line-buffer          --recstart '>'         --cat          run     < in.fasta     | cat > out.txt
  
     predutils r2js       --pipeline-version "1.1.0-alpha"       signalp6 "out.txt"     > "out.ldjson"

Command exit status:
  137

Command output:
  (empty)

Command error:
  WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
  Matplotlib created a temporary config/cache directory at tmp162/matplotlib-hbbo0rts because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
  Matplotlib created a temporary config/cache directory at tmp163/matplotlib-5dprkzdr because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
  Matplotlib created a temporary config/cache directory at tmp158/matplotlib-qvjqlyvk because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
  Matplotlib created a temporary config/cache directory at tmp156/matplotlib-1ryo928c because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
  
  
  
  
  Predicting:   0%|          | 0/100 [00:00<?, ?batch/s]/opt/conda/envs/predector/lib/python3.7/site-packages/torch/nn/modules/module.py:1051: UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at  /tmp/pip-req-build-xi8rb33b/aten/src/ATen/native/TensorCompare.cpp:255.)
    return forward_call(*input, **kwargs)
  
  Predicting:   0%|          | 0/100 [00:00<?, ?batch/s]environment: line 1:   169 Killed                  signalp6 --fastafile "$1" --output_dir "${TMPDIR}" --format txt --organism eukarya --mode fast 1>&2
  parallel: This job failed:
  run $PARALLEL_TMP

Work dir:
  /ppgdata/fredrick/assembly_data/ascochyta/FINAL_Assemblies/genome_annotation/work/1d/86a3226589c58fd35a4b5e72410b3c

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
```

**Comment by fmobegi** [comment]
Fixed with `1.1.0-beta`

**Comment by darcyabjones** [comment]
Thanks! Sorry for the bug
